src/runtime/virtcontainers/sandbox.go

```diff
@@ -482,6 +484,7 @@ func createSandbox(ctx context.Context, sandboxConfig SandboxConfig, factory Fac
 
 func newSandbox(ctx context.Context, sandboxConfig SandboxConfig, factory Factory) (sb *Sandbox, retErr error) {
 	span, ctx := trace(ctx, "newSandbox")
+	span.SetAttributes(otelLabel.String("sandbox_config_id", sandboxConfig.ID), otelLabel.String("subsystem", "sandbox"))
```

Comment:
Does `sandbox_config_id` equal `sandbox_id` ? So should use `sandbox_id` ?

Reply:
@liubin yes, fixed